ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/DdlCommandExec.java

```diff
@@ -32,36 +31,15 @@ public DdlCommandExec(final MetaStore metaStore) {
     this.metaStore = metaStore;
   }
 
-  /**
-   * execute on temp metaStore
-   */
-  public DdlCommandResult tryExecute(final DdlCommand ddlCommand, final MetaStore tempMetaStore) {
```

Comment:
no longer needed as this functionality is replicated by having a different ExecutionContext within the engine.